### ISSUE TITLE
Update "Start My Server" button

### ISSUE THREAD

**Issue by pierrepo** [plasmabio/tljh-repo2docker#38]
Following @psychemedia comment in issue #37 

> A UI note... If you stop the server, then you are presented with a Start Server button. My expectation is that that would (re)start the server you just stopped, rather than take you to the start server selection page. Tweaking the button text to Select and start server would make things clearer?

It would indeed be interesting to rename the "Start My Server" button to "Select and Start My Server". 

Ideally, we should also skip the "Launch Server" page to jump directly to the "Server Option" page.

**Comment by psychemedia** [comment]
> Ideally, we should also skip the "Launch Server" page to jump directly to the "Server Option" page.

Yes that was my initial thought, although other possibilities suggest themselves:

1) you offer a "restart previous server [description]" button as well as a link to "select and start server" page;
2) the select page highlights the server you previously started in some way;

**Comment by jtpio** [comment]
This might require a change to the JupyterHub base template, so this part of the home page becomes more configurable:

https://github.com/jupyterhub/jupyterhub/blob/0317fd63fa1dac4d709b4f95738e74c5c408da6c/share/jupyterhub/templates/home.html#L7-L20
